src/azul/plugins/metadata/anvil/service/response.py

```diff
@@ -210,6 +210,7 @@ def _non_pivotal_fields_by_entity_type(self) -> dict[str, set[str]]:
             },
             'datasets': {
                 'dataset_id',
+                'duos_id',
```

Comment:
Do you know why `description` isn't listed here?

Reply:
This list is in `_non_pivotal_fields_by_entity_type`. In non-datasets endpoints, the `datasets` inner entity only contains a small subset of `datasets` fields.

```
$ http 'https://service.explore.anvilproject.org/index/activities?size=2' | jq '.hits[].datasets'
[
  {
    "dataset_id": [
      "8a756d17-c54c-3b33-312b-c767552a8516"
    ],
    "title": [
      "ANVIL_1000G_high_coverage_2019"
    ]
  }
]
[
  {
    "dataset_id": [
      "8a756d17-c54c-3b33-312b-c767552a8516"
    ],
    "title": [
      "ANVIL_1000G_high_coverage_2019"
    ]
  }
]
```